sites/default/sqlconf.php

```diff
@@ -1,31 +0,0 @@
-<?php
```

Comment:
Dude...can't delete the sqlconf.php file!  (not yet anyway!)  Should have been a gitignore, if you are running your test instance from your repo.
  If you like, you could make a mechanism that generates this file at setup...but you got to put this back before we can merge it.